src/preferences/PreferencesBase.js

```diff
@@ -173,6 +174,14 @@ define(function (require, exports, module) {
                 prefFile.read({}, function (err, text) {
                     if (err) {
                         if (createIfNew) {
```

Comment:
Is the `createIfNew` flag safe to use? Should it check explicitly for `state.json`? Seems like we'd want to handle `brackets.json` the same way. What about project pref files?